actions/lib/create_archive_dir.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+
+import os, sys, shutil, argparse
+
+def verify_src(srcdir, threshold = None):
+  if not os.path.exists(srcdir) or not os.path.isdir(srcdir):
+    print("Runfolder {} is not a directory and/or doesn't exist. Aborting.".format(srcdir))
+    sys.exit(1)
+
+  unaligned_link = os.path.join(srcdir, "Unaligned")
+  unaligned_dir = os.path.abspath(unaligned_link)
+
+  if not os.path.exists(unaligned_link) or not os.path.islink(unaligned_link): 
+    print("Expected link {} doesn't seem to exist or is broken. Aborting.".format(unaligned_link))
+    sys.exit(1)
+  elif not os.path.exists(unaligned_dir) or not os.path.isdir(unaligned_dir): 
+    print("Expected directory {} doesn't seem to exist. Aborting.".format(unaligned_dir))
+    sys.exit(1) 
+  else:
+    counter = 0
+
+    for root, dirs, files in os.walk(unaligned_dir):
+      for file in files: 
+        if file.lower().endswith(".fastq.gz"):
+          counter = counter + 1
+  
+    if threshold and counter < threshold: 
+        print("We found only {} files that seems to be FASTQ files. Expecting at least {} files. Looks suspicious; aborting.".format(counter, threshold))
+        sys.exit(1)
+
+def verify_dest(destdir, remove=False): 
+  if os.path.exists(destdir):
+    if remove: 
+      print("Archive directory {} already exists. Operator requested to remove it.".format(destdir))
+      shutil.rmtree(destdir)
+    else: 
+      print("Archive directory {} already exists. Aborting.".format(destdir))
+      sys.exit(1)
+
+def create_dest(srcdir, destdir, exclude = None): 
+  os.makedirs(destdir)
```

Comment:
In the args parse docs it says that the destination will be overwritten if it exits, but from what I can tell from this: https://docs.python.org/2/library/os.html#os.makedirs it will not be. Rather it will raise an error if the destination exits.

However since you first verify the destination and use an argument to remove it if that has been specified, I guess this should be fine, and only the docs need to be updated to reflect this?